assets/sidecars/volumesnapshot_provisioner_role.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: volumesnapshot-external-provisioner-role
+rules:
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshots"]
+    verbs: ["get", "list"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshotcontents"]
+    verbs: ["get", "list"]
```

Comment:
dtto, if a storage backed supports snapshots, this (and more) will be part of external-snapshotter role.

Reply:
Same as in the comment above: provisioner needs only two verbs: `["get", "list"]`, while snapshotter wants more: `["get", "list", "watch", "update"]`.